docs/security.md

```diff
@@ -130,21 +130,16 @@ authentication:
 
 ## Authorization
 
-Trino Gateway supports the following roles in regex string format:
+Trino Gateway supports the following roles in regex string format (for API calls):
```

Comment:
why .. those groups are supported by the UI as well 

```suggestion
Trino Gateway supports the following roles in regex string format:
```

Reply:
and if not .. thats a bug in the UI code

Reply:
You would need to refactor the `hasRole` call to get the UI and the web API to function the same.

https://github.com/trinodb/trino-gateway/blob/739ec5e67799dd5df0ab8aaec7934e21a63507fa/webapp/src/store/access.ts#L73C9-L76C9

See the following places where it is used:

https://github.com/trinodb/trino-gateway/blob/ac407dcbd02c3e00655a2ce0a63dd21be429585b/webapp/src/components/selector.tsx#L80
https://github.com/trinodb/trino-gateway/blob/ac407dcbd02c3e00655a2ce0a63dd21be429585b/webapp/src/components/cluster.tsx#L89
https://github.com/trinodb/trino-gateway/blob/ac407dcbd02c3e00655a2ce0a63dd21be429585b/webapp/src/components/history.tsx#L93
https://github.com/trinodb/trino-gateway/blob/ac407dcbd02c3e00655a2ce0a63dd21be429585b/webapp/src/components/resource-group.tsx#L88